### DISCUSSION TITLE
optimal approach for library free DIA when there is just a subset of proteins heavily modified

### DISCUSSION THREAD

**Discussion by weiclav** [vdemichev/DiaNN#666] [Q&A] [answered]
Hi Vadim,

I just would like to double check with you what would be the optimal approach for the library free DIA processing when you have samples measured in DIA where just a subset of proteins is heavily modified where omitting the modification may lead to substantial loss of high intense precursors or even high abundant proteins (e.g. tissue scaffold samples with collagen proteins rich in proline oxidation modification).

I have simply put all proteins (collagen proteins in our case) where the modification is expected aside from the rest of the human proteome and created two libraries using the two proteins set - one with no variable modification for proteins where the modification is not expected to be that prominent (all but collagen proteins in our case) and the other one for the potentially modified proteins (collagen proteins in our case). Libraries were build using the same settings but the variable modification part of course (in our case oxidation of P set as variable, with up to 3 variable modifications set and --strip-unknown-mods was used in the latter case). Both libraries were converted to the tsv format and used for the library free search together. In the data processing step using the two libraries, the modifications were set based on the library generated for the modified proteins (i.e. variable P oxidation was set, up to 3 variable mods and --strip-unknown-mods extra option), all other settings were the ones I would use normally.

The results look as expected, there is slight improvement when this approach is used compared to the situation when I would not use oxidation of P at all or set it as variable for the whole proteome. I am just not 100% sure it is all correct regarding the FDR estimates etc. It should be more appropriate I think as you are not missing anything high intense in the data so the NN should work better and there is less likely chance of some false positives but not sure how DIANN process the two libraries having different modifications setup internally... So, better asking for your advice and green light to use this approach :).

Thank you a lot for your insight and once more for the great tool!

Best,
David

**Comment by vdemichev** [comment, marked as answer]
Hi David,

Sounds good!
Some comments:
- strip-unknown-mods only applies to in silico prediction, don't need this for actual searching with pre-made libraries
- you might want to see if you want to use --monitor-mod on your modification
- petidoform FDRs should be perfectly fine with --moninor-mod, regular FDR is fine already

Best,
Vadim

**Reply by weiclav** [reply]
Great, thx a lot for the lightning fast response, really do appreciate that!

Will double check the proposed options as well!

Take care!
David
